packages/client/test/rpc/debug/traceCall.spec.ts

```diff
@@ -0,0 +1,128 @@
+import { Block } from '@ethereumjs/block'
+import { TransactionFactory } from '@ethereumjs/tx'
+import { bytesToHex } from '@ethereumjs/util'
+import { assert, describe, expect, expectTypeOf, it } from 'vitest'
+
+import genesisJSON from '../../testdata/geth-genesis/debug.json'
+import {
+  baseRequest,
+  createClient,
+  createManager,
+  dummy,
+  params,
+  runBlockWithTxs,
+  setupChain,
+  startRPC,
+} from '../helpers'
+
+const method = 'debug_traceCall'
+
+describe(method, () => {
+  const manager = createManager(createClient({ opened: true }))
+  const methods = manager.getMethods()
+  const server = startRPC(methods)
+
+  it(' debug_traceCall method exists', async () => {
```

Comment:
Seems like there is an unnecessary leading space, here:
```suggestion
  it('debug_traceCall method exists', async () => {
```